raiden-dapp/src/locales/en.json

```diff
@@ -180,6 +183,7 @@
   "no-access": {
     "unsupported-network": "The current network is unsupported. Please choose a different network.",
     "sdk-initialization-failure": "SDK initialization failed. Please check the console for more information.",
+    "rdn-state-migration": "Cannot replace state with older uploaded state.",
```

Comment:
In the future we should get rid of these and use the `ErrorMessage` component to provide some more info on these errors. As a user it's really difficult to understand how they can get out of their misery